scenarioo-client/protractor-e2e-scenarioo.conf.js

```diff
@@ -30,16 +31,17 @@ var exportsConfig = {
     directConnect: true,
 
     // Timeouts: https://angular.github.io/protractor/#/timeouts
-    allScriptsTimeout: 20000,
-    getPageTimeout: 20000,
+    allScriptsTimeout: 5000,
+    getPageTimeout: 5000,
```

Comment:
Achung! Die Timeouts müssen so gewählt werden, dass die Tests auch noch lokal wie auch auf dem CI (Jenkins und Circle CI) am Schluss zuverlässig grün sind! 5000 für all scripts vermutlich etwas kurz - page timeout sollte 5 s locker reichen (war vorher sehr hoch) - lieber nur 2-3 s

Reply:
Changed it to 8 and 2 seconds